src/misc/cross.cpp

```diff
@@ -439,6 +440,224 @@ FILE *fopen_wrap_ro_fallback(const std::string &filename, bool &is_readonly)
 	return fp;
 }
 
+bool wild_match(const char *haystack, char *needle)
+{
+	size_t max, i;
+	for (; *needle != '\0'; ++needle) {
```

Comment:
If `needle` is null this `*needle` will crash.
At a minimum, lets assert(haystack) and assert(needle).

Reply:
Done for both.